x-pack/plugins/apm/public/components/app/service_overview/service_overview_throughput_chart.tsx

```diff
@@ -123,32 +117,9 @@ export function ServiceOverviewThroughputChart({
                 'xpack.apm.serviceOverview.throughtputChartTitle',
                 { defaultMessage: 'Throughput' }
               )}
-              {data.throughputUnit === 'second'
-                ? i18n.translate(
-                    'xpack.apm.serviceOverview.throughtputPerSecondChartTitle',
-                    { defaultMessage: ' (per second)' }
-                  )
-                : ''}
             </h2>
           </EuiTitle>
         </EuiFlexItem>
-
-        <EuiFlexItem grow={false}>
-          <EuiIconTip
-            content={
-              data.throughputUnit === 'minute'
-                ? i18n.translate('xpack.apm.serviceOverview.tpmHelp', {
-                    defaultMessage:
-                      'Throughput is measured in transactions per minute (tpm)',
-                  })
-                : i18n.translate('xpack.apm.serviceOverview.tpsHelp', {
-                    defaultMessage:
-                      'Throughput is measured in transactions per second (tps)',
-                  })
-            }
-            position="right"
-          />
-        </EuiFlexItem>
```

Comment:
Removing this now makes it harder to re-add later doesn't it? Why not leave it in an simply hardcode `minute` as throughput unit on the server?

Reply:
why would it be harder to re-add it later?

Reply:
If we don't remove it now we don't have to re-add it later.